eli5/formatters/utils.py

```diff
@@ -68,3 +68,8 @@ def _has_invisible_spaces(name):
     if isinstance(name, list):
         return any(_has_invisible_spaces(n['name']) for n in name)
     return name.startswith(' ') or name.endswith(' ')
+
+
+def max_or_0(it):
```

Comment:
I think eli5.utils is a better place for it; it is a general-purpose function not related to formatters directly. Could you please also add a test for it (a doctest is fine)?